### PR TITLE
add LLD ques

### PR DESCRIPTION
**Pull request by babitarit**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1836,7 +1836,17 @@ const generalCSQuiz = [
     Explanation:
       "The Halting Problem is significant in theoretical computer science as it demonstrates the existence of problems that cannot be solved algorithmically.",
     Link: "https://en.wikipedia.org/wiki/Halting_problem"
-  }
+  },
+  {
+    Question: "What is load Balancing in LLD(Low Level Design)?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1841]:
```suggestion
    Question: "What is load balancing in LLD(Low Level Design)?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by babitarit** [comment]
@jdwilkin4 i had done the required changes please review it

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**